quinn-h3/src/lib.rs

```diff
@@ -95,27 +99,52 @@ impl From<quinn::ConnectionError> for Error {
     }
 }
 
+impl From<ErrorCode> for Error {
+    fn from(code: ErrorCode) -> Self {
+        match code {
+            ErrorCode::REQUEST_CANCELLED => Error::RequestCancelled,
+            ErrorCode::REQUEST_REJECTED => Error::RequestRejected,
+            _ => panic!("wut"),
```

Comment:
Maybe make the `panic!()` message a little more descriptive here? Or otherwise use `unreachable!()` or `unimplemented!()` or some such.

Reply:
Oh, sure. (at least this gave me a good laugh)